index.html

```diff
@@ -50,6 +52,250 @@ <h1>Hey there. I’m Mariane<br />
             </div>
         </div>
     </section>
+    <section class='recent-works'>
+        <div class="wrapper">
+            <div class="dropdown"></div>
+            <h2>My Recent Works</h2>
+            <span class="bottom"></span>
+
+            <div class="allprojects-container">
+                <div class="project-container-card">
+                    <div class="project-image"></div>
+                    <div class="project-desc-tech-details">
+                        <h4 class="project-desc">Multi-Post Stories<br /> Gain + Glory</h4>
+                        <ol class="tech-list-container">
+                            <li class="tech-list">
+                                <p class="tech-name">Ruby on Rails</p>
+                            </li>
+                            <li class="tech-list">
+                                <p class="tech-name">css</p>
+                            </li>
+                            <li class="tech-list">
+                                <p class="tech-name">JavaScript</p>
+                            </li>
+                            <li class="tech-list">
+                                <p class="tech-name">html</p>
+                            </li>
+                        </ol>
+                        <button class="btn btn-projects" type="button">See Projects</button>
+                    </div>
+                </div>
+                <div class="project-container-card">
+                    <div class="project-image"></div>
+                    <div class="project-desc-tech-details">
+                        <h4 class="project-desc">Multi-Post Stories<br /> Gain + Glory</h4>
+                        <ol class="tech-list-container">
+                            <li class="tech-list">
+                                <p class="tech-name">Ruby on Rails</p>
+                            </li>
+                            <li class="tech-list">
+                                <p class="tech-name">css</p>
+                            </li>
+                            <li class="tech-list">
+                                <p class="tech-name">JavaScript</p>
+                            </li>
+                            <li class="tech-list">
+                                <p class="tech-name">html</p>
+                            </li>
+                        </ol>
```

Comment:
## BEAUTIFUL DESIGN 😊👍
- [ ] Good job so far, unfortunately, some issues need to be done 🥲. Please change all your `<ol>` tags to `<ul>`. 

These tags should be fixed 👍
<img width="326" alt="Screen Shot 2022-01-15 at 3 04 19 PM" src="https://user-images.githubusercontent.com/69011963/149618015-a7325bba-4b9d-45a3-a17f-9a313724ea20.png">

Reply:
Well implemented 👍🏽

Reply:
Thanks @Rustamxon7 ,I have changed all affected tags as suggested